blueprint: Various blueprint clean-ups

kayousterhout opened the pull request.

luise commented on django-example.js:
```diff
@@ -16,12 +16,13 @@ deployment.deploy(baseMachine.asWorker().replicate(3));
 
 // Create applications.
 const mongo = new Mongo(3);
+mongo.deploy(deployment);
```
Ugh, it's ugly to sometimes have `deployment.deploy` and at other times have `.deploy(deployment)`. Not very intuitive. We should keep that in mind, when we redesign `createDeployment`

kayousterhout replied:
Ah it's a good point that our proposal last week for re-doing the deployment "constructor" would fix this issue  (https://github.com/quilt-corp/infra/pull/7/files#diff-e51680e2be5572bebce0ca9da574b64aR22)

luise replied:
Yes, that would be another nice benefit!

kayousterhout commented:
I made two changes to this PR: I changed the deploy() commit to also use container.deploy() directly, and I added a commit that changes the naming to camelCase for consistency with other repos (note that this does not require a re-build of the Dockerfile that's in Quilt)
